onnx_chainer/export.py

```diff
@@ -137,17 +137,19 @@ def create_node(
     def backward_postprocess(self, function, in_data, out_grad):
         if isinstance(function, chainer.function.FunctionAdapter):
             function = function.function
-        func_name = function.__class__.__name__
+        func_name = getattr(function, 'user_name', function.__class__.__name__)
```

Comment:
I feel `user_name` sounds a bit awkward, I don't have a great idea though. `user_defined_name` or `custom_function_node_name` would be slightly better? I think using a longer name for this field would be a good idea. There would be less chance of having a real FunctionNode with `custom_function_node_name` than one with `user_name`.

Reply:
adopted `custom_function_node_name`, thx